core/src/test/scala/integration/kafka/server/RaftClusterTest.scala

```diff
@@ -29,7 +29,7 @@ import java.util
 import java.util.Collections
 import scala.jdk.CollectionConverters._
 
-@Timeout(120000)
+@Timeout(120)
```

Comment:
Hmm, I don't think 0.12 seconds is a good timeout for these integration tests. I would be OK cutting the time down to 60 seconds per test, though, rather than 120.

Reply:
I think the default unit is seconds, so this is 120s.